php/Terminus/Dispatcher/Subcommand.php

```diff
@@ -166,10 +167,10 @@ private function validate_args( $args, $assoc_args, $extra_args ) {
 
     $cmd_path = implode( ' ', get_path( $this ) );
     foreach ( $validator->get_unknown() as $token ) {
-      \Terminus::warning( sprintf(
-        "The `%s` command has an invalid synopsis part: %s",
-        $cmd_path, $token
-      ) );
+      \Terminus::get_logger()->warning(
```

Comment:
Well, yes and no. We are targeting PHP 5.4+ for 1.0.0, but this syntax was only added in 5.4 (if memory serves), and when we'd added the ()-> syntax before, several big clients complained that Terminus was totally broken for them. :(

To be clear about how I envisioned this working, I believed that we are enforcing PHP 5.4+ *starting with* 1.0.0, that is to say:
**Supports PHP 5.3+:** ...0.7.0 -bleeding edge up to-> 0.7.1 -bleeding edge up to-> 0.8.0 -bleeding edge up to->
**Supports PHP 5.4+:** 1.0.0 -bleeding edge up to-> 1.1.0, etc

Do you think that the 5.4+ column should have the bleeding edge up to 1.0.0, too? If so, should we change the version check on php/terminus.php to prevent earlier versions from running it?